crates/spirv-builder/src/test/basic.rs

```diff
@@ -200,29 +200,6 @@ OpDecorate %4 Binding 0
     );
 }
 
-// NOTE(eddyb) we specifically run Vulkan validation here, as the default
-// validation rules are more lax and don't require a `Block` decoration
-// (`#[spirv(block)]` here) on `struct ShaderConstants`.
-#[test]
-fn push_constant_vulkan() {
-    val_vulkan(
```

Comment:
Can `val_vulkan` be removed now? There's some hacks IIRC, between `rustc_codegen_spirv` and `spirv-builder`, just to make it work.